addons/account/models/sequence_mixin.py

```diff
@@ -128,3 +160,4 @@ def _set_next_sequence(self):
         )
         self[self._sequence_field] = value
         self.flush([self._sequence_field])
+        self._compute_split_sequence()
```

Comment:
why is a manual call to `_compute_split_sequence` needed, since you manually added the dependency between the function and the field? :suspect:

Reply:
It needs to be computed **and flushed** right away, because we search in the db to compute the following `self[self._sequence_field]`, needing those fields.

Reply:
Not needed with the correct `@api.depends`

Reply:
What does' the `@api.depends` have to do with the fact that it works? Isn't it strictly the same as declaring it in the `__init__` function?
Either way, I tried to remove it, and it doesn't work.